.github/workflows/update_lite_galata_references.yaml

```diff
@@ -0,0 +1,111 @@
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get PR Info
+        id: pr
+        env:
+          PR_NUMBER: ${{ github.event.issue.number }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_REPO: ${{ github.repository }}
+          COMMENT_AT: ${{ github.event.comment.created_at }}
+        run: |
+          pr="$(gh api /repos/${GH_REPO}/pulls/${PR_NUMBER})"
+          head_sha="$(echo "$pr" | jq -r .head.sha)"
+          pushed_at="$(echo "$pr" | jq -r .pushed_at)"
+
+          if [[ $(date -d "$pushed_at" +%s) -gt $(date -d "$COMMENT_AT" +%s) ]]; then
+              echo "Updating is not allowed because the PR was pushed to (at $pushed_at) after the triggering comment was issued (at $COMMENT_AT)"
+              exit 1
+          fi
+
+          echo "head_sha=$head_sha" >> $GITHUB_OUTPUT
+
+      - name: Checkout the branch from the PR that triggered the job
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: gh pr checkout ${{ github.event.issue.number }}
+
+      - name: Validate the fetched branch HEAD revision
+        env:
+          EXPECTED_SHA: ${{ steps.pr.outputs.head_sha }}
+        run: |
+          actual_sha="$(git rev-parse HEAD)"
+
+          if [[ "$actual_sha" != "$EXPECTED_SHA" ]]; then
+              echo "The HEAD of the checked out branch ($actual_sha) differs from the HEAD commit available at the time when trigger comment was submitted ($EXPECTED_SHA)"
+              exit 1
+          fi
+
+      - name: Download extension package
+        uses: dawidd6/action-download-artifact@v6
+        with:
+          pr: ${{github.event.issue.number}}
+          workflow: 'build.yml'
+          workflow_conclusion: ''
+          name: lite-artifacts
+          path: dist
+          allow_forks: true
+
+      # - name: Install Conda environment with Micromamba
+      #   uses: mamba-org/setup-micromamba@v1
+      #   with:
+      #     environment-name: cad
+      #     create-args: >-
+      #       python=3.9
+      #       jupyterlab=4
+      #       pythonocc-core=7.7.0
+
+      # - name: Install the extension
+      #   shell: bash -l {0}
+      #   run: |
+      #     whereis python
+      #     cp ./jupytercad_core/dist/jupytercad*.whl ./jupytercad_lab/dist/jupytercad*.whl ./jupytercad_app/dist/jupytercad*.whl ./jupytercad/dist/jupytercad*.whl .
+      #     python -m pip install jupytercad*.whl
+
+      - name: Install dependencies
+        shell: bash -l {0}
+        working-directory: ui-tests
+        env:
+          PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
+        run: |
+          jlpm install
+          yarn playwright install
+
+      - uses: jupyterlab/maintainer-tools/.github/actions/update-snapshots@main
+        with:
+          npm_client: jlpm
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          start_server_script: 'null'
+          test_folder: ui-tests
```

Comment:
I believe you're missing an `update_script` parameter here https://github.com/jupyterlab/maintainer-tools/blob/main/.github/actions/update-snapshots/action.yml#L58

We need a script that updates the lite snapshots specifically

Reply:
Does [`8e0ffaf`](https://github.com/jupytercad/JupyterCAD/pull/696/commits/8e0ffaf9b23cc2bd5e6929b4222244a7a9785832) does it correctly?